### ISSUE TITLE
[Feature Request] Option for adding "[FreeLeech]" tag to freeleech torrent results. 

### ISSUE THREAD

**Issue by batuhankrmn** [Jackett/Jackett#11707]
I use Sonarr with Jackett integration and I appreciate both software for what amazing job they do. I prefer freeleech torrents but "search only freeleech" settings is not what I mean. I download non-freeleech ones too when I need. Since Sonarr doesnt have option as "prefer freeleech torrents" like Radarr, I though If we can add the "[FreeLeech]" tag to the freeleech ones among the torrent results, so this [FreeLeech] tag can be added to preferred words in Sonarr and FreeLeech ones will be preferred in that way. I think this is doable because qBitorrent search engine is already doing this with adding tracker name to results. Adding this feature as an option would be great.

Example Format:
[Idope] Joker.2019.2160p.UHD.BluRay.x265-AAAUHD [FreeLeech]

**Comment by batuhankrmn** [comment]
Yeah Radarr handles "prefer freeleech torrents" option with custom formats feature which is not supported by Sonarr yet. Maybe they will provide in the future but I thought it would not be difficult to do this via Jackett since It already takes the freeleech infos from trackers. 

Additionally, I've used qBittorrent-Jackett combo for a long time before I use sonarr/radarr and I was always hesitant when I would download torrents from torrent sites with strict ratio rules. I always had to right click and click "open description page" to check If It's freeleech or not. 

If this feature is added as optionally, I think it will be very useful for users other than Sonarr / Radarr too. In this way, those who want can enable this feature.

**Comment by garfield69** [comment]
While It may be a simple matter to add this feature to Jackett, it is not a trivial process to implement.
Each individual indexer is responsible for how it processes the query it makes of the web site and the returned results.
That is over 500 individual indexers that have to be amended.
Lets see if we get any feedback from other team members or interest from other Jackett users.

**Comment by batuhankrmn** [comment]
Of course! The Jackett system is working very well right now. I don't want to tire this out with an overly personal request, but I truly believe this will be a nice feature for the vast majority. Thanks for considering!

**Comment by ilike2burnthing** [comment]
I'd agree with garfield here, if Radarr is already doing it then Sonarr is perfectly capable of supporting it too. In saying that, this was [requested 2.5yrs ago](https://github.com/Sonarr/Sonarr/issues/2782), and the ability to do it was [added nearly 5yrs ago](https://github.com/Sonarr/Sonarr/pull/1464), so...

**Comment by batuhankrmn** [comment]
> Yeah, Jackett should definitely not implement modifying the release title like that. This is exactly why we added these attributes to torznab in the first place.
> Regretfully it's not been implemented in Sonarr (despite the elapsed time). There has been some work done in a PR for indexer attributes/flags a while back by a contributor, which is related to this as it exposes a number of flexible attributes to the Release Profile syntax.

Since Sonarr developer is here and saying this should not be done, our best bet is waiting Sonarr to provide this feature.

But I also want to mention that this feature is one of the best features Radarr provides, and it can do this thanks to Custom Format.  I wonder if Sonarr needs to make a radical change to achieve this (like adding support for Custom Formats) , or can this be achieved with minor updates?
